CHANGELOG.md

```diff
@@ -3,6 +3,70 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.16.3
+
+_Oct 20, 2023_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@6.16.3`
+
+- [DataGrid] Allow passing readonly arrays to `columns` and `sortingOrder` props (#10686) @pcorpet
+
+#### `@mui/x-data-grid-pro@6.16.3` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@6.16.3`.
+
+
+#### `@mui/x-data-grid-premium@6.16.3` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@6.16.3`.
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@6.16.3`
+
+- [fields] Correctly respect leading zeroes on seconds section (#10713) @flaviendelangle
+- [fields] Use `onChange` instead of `onKeyPress` for Backspace editing (#10494) @flaviendelangle
+- [pickers] Add reference links to DatePicker components (#10626) @michelengelen
+- [pickers] Add reference links to clock components (#10645) @michelengelen
+- [pickers] Add reference links to misc picker components (#10647) @michelengelen
+- [pickers] Add reference links to toolbar components (#10646) @michelengelen
+- [pickers] Change the props received by the `FakeTextField` component (#10687) @flaviendelangle
+
+#### `@mui/x-date-pickers-pro@6.16.3` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@6.16.3`, plus:
+
+- [DateRangePicker] Fix touch based range dragging (#10664) @michelengelen
+
+### Charts / `@mui/x-charts@__CHARTS_VERSION__`
+
+- [charts] Add reference links to area + bar chart components (#10652) @michelengelen
+- [charts] Add reference links to line chart + sparkline components (#10650) @michelengelen
+- [charts] Add reference links to pie + scatter chart components  (#10653) @michelengelen
+- [charts] Skip rendering if not width or height (#10714) @alexfauquette
+- [charts] Support animation (#9926) @alexfauquette
```

Comment:
I feel like this deserves a highlight 💯 WDYT?

Reply:
Clearly!

Reply:
Maybe we could also be more explicit? 🤔 
```suggestion
- [charts] Support animation on `BarChart` (#9926) @alexfauquette
```